compiler/rustc_span/src/fatal_error.rs

```diff
@@ -1,6 +1,4 @@
-/// Used as a return value to signify a fatal error occurred. (It is also
```

Comment:
the panic!() comment refers to 2015 panic, which is panic_any

Reply:
Makes sense. I have updated the commit accordingly.